src/language-js/parse/babel.js

```diff
@@ -105,7 +105,7 @@ function createParse(parseMethod, ...optionsCombinations) {
       }));
     }
 
-    if (text.includes("#{") || text.includes("#[")) {
+    if (/#({|\[)/.test(text)) {
```

Comment:
```suggestion
    if (/#[[{]/.test(text)) {
```

Reply:
I'm suprised that `eslint-plugin-import` didn't report

Reply:
I want to review the PR, but I'm not familiar with regex sorry